os/board/rtl8721csm/src/component/os/os_dep/include/osdep_service.h

```diff
@@ -179,7 +179,7 @@ typedef int (*event_handler_t)(char *buf, int buf_len, int flags, void *user_dat
 #define CONFIG_THREAD_COMM_SEMA
 struct task_struct {
 	const char *task_name;
-	_thread_hdl_ task;	/* I: workqueue thread */
+	pid_t task;	/* I: workqueue thread */

```

Comment:
Because you change this type, you should check the code which use this variable like the tid in _tizenrt_create_task function. 
```
	pthread_t *tid = _tizenrt_zmalloc(sizeof(pthread_t));
	if (tid == NULL) {
		DBG_ERR("Failed to malloc for tid\n");
		goto err_exit;
	}
	res = pthread_create(tid, &attr, (pthread_startroutine_t) func, thctx);
==>
	pthread_t tid;
	res = pthread_create(&tid, &attr, (pthread_startroutine_t) func, thctx);
```

Reply:
```
	ptask->task = NULL;
==>
	ptask->task = -1;
```

Reply:
Sure, changed as adviced.